diff --git a/src/scripts/Makefile b/src/scripts/Makefile
index 9d4e22fee..626683c4b 100644
--- a/src/scripts/Makefile
+++ b/src/scripts/Makefile
@@ -503,7 +503,7 @@ LIBCONV_CORE=convcore.o conv-conds.o conv-taskQ.o queueing.o msgmgr.o \
 	converseProjections.o machineProjections.o \
 	quiescence.o isomalloc.o mem-arena.o conv-counter.o memory-darwin-clang.o \
 	global-nop.o cmipool.o cpuaffinity.o cputopology.o  \
-	cmitls.o memoryaffinity.o commitid.o conv-interoperate.o conv-rdma.o \
+	memoryaffinity.o commitid.o conv-interoperate.o conv-rdma.o \
 
 LIBCONV_LDB = topology.o generate.o edgelist.o
 
