diff --git a/packages/seacas/applications/exodiff/iqsort.C b/packages/seacas/applications/exodiff/iqsort.C
index 4bf90ecb70..350a8049f9 100644
--- a/packages/seacas/applications/exodiff/iqsort.C
+++ b/packages/seacas/applications/exodiff/iqsort.C
@@ -31,6 +31,7 @@
 // OF THIS SOFTWARE, EVEN IF ADVISED OF THE POSSIBILITY OF SUCH DAMAGE.
 //
 #include "iqsort.h"
+#include <cstdint>
 
 namespace {
   template <typename INT> void swap_(INT v[], size_t i, size_t j);
