diff --git a/packages/teuchos/comm/src/Teuchos_PerformanceMonitorBase.hpp b/packages/teuchos/comm/src/Teuchos_PerformanceMonitorBase.hpp
index 6eb0b815a5..fd2bf76bf8 100644
--- a/packages/teuchos/comm/src/Teuchos_PerformanceMonitorBase.hpp
+++ b/packages/teuchos/comm/src/Teuchos_PerformanceMonitorBase.hpp
@@ -222,7 +222,7 @@ namespace Teuchos
         // out of memory for storing callbacks).  We could throw an
         // exception here in that case, but I think it's better just
         // to let the minor memory leak happen.
-        (void) atexit (freeTableFormat);
+        static_cast<void>(atexit(freeTableFormat));
       }
       TEUCHOS_TEST_FOR_EXCEPTION(
         format_ == NULL, std::logic_error, "Teuchos::PerformanceMonitorBase::"
@@ -305,7 +305,7 @@ namespace Teuchos
         // out of memory for storing callbacks).  We could throw an
         // exception here in that case, but I think it's better just
         // to let the minor memory leak happen.
-        (void) atexit (freeCounters);
+        static_cast<void>(atexit(freeCounters));
       }
       TEUCHOS_TEST_FOR_EXCEPTION(
         counters_ == NULL, std::logic_error, "Teuchos::PerformanceMonitorBase::"
diff --git a/packages/zoltan/src/coloring/color_test.c b/packages/zoltan/src/coloring/color_test.c
index 465a6a97ed..47fca6fa94 100644
--- a/packages/zoltan/src/coloring/color_test.c
+++ b/packages/zoltan/src/coloring/color_test.c
@@ -59,7 +59,7 @@ extern "C" {
 #include "g2l_hash.h"
 #include "params_const.h"
 #include "zz_util_const.h"
-#include "graph.h"
+#include "../graph/graph.h"
 #include "all_allo_const.h"
 
 
diff --git a/packages/zoltan/src/coloring/coloring.c b/packages/zoltan/src/coloring/coloring.c
index edef3fecf2..591a4892f2 100644
--- a/packages/zoltan/src/coloring/coloring.c
+++ b/packages/zoltan/src/coloring/coloring.c
@@ -56,7 +56,7 @@ extern "C" {
 #include "g2l_hash.h"
 #include "params_const.h"
 #include "zz_util_const.h"
-#include "graph.h"
+#include "../graph/graph.h"
 #include "all_allo_const.h"
 #include "zz_rand.h"
 #include "bucket.h"
diff --git a/packages/zoltan/src/tpls/build_graph.c b/packages/zoltan/src/tpls/build_graph.c
index 6a66a64c44..09f2831619 100644
--- a/packages/zoltan/src/tpls/build_graph.c
+++ b/packages/zoltan/src/tpls/build_graph.c
@@ -54,7 +54,7 @@ extern "C" {
 #include "zz_util_const.h"
 #include "all_allo_const.h"
 #include "params_const.h"
-#include "graph.h"
+#include "../graph/graph.h"
 
 int Zoltan_Build_Graph(ZZ *zz, int *graph_type, int check_graph,
 		       int num_obj, ZOLTAN_ID_PTR global_ids, ZOLTAN_ID_PTR local_ids,
diff --git a/packages/zoltan/src/tpls/third_library.h b/packages/zoltan/src/tpls/third_library.h
index 6ea4b56351..070c98bd1c 100644
--- a/packages/zoltan/src/tpls/third_library.h
+++ b/packages/zoltan/src/tpls/third_library.h
@@ -51,7 +51,7 @@
 #include <limits.h>
 #include "zoltan_comm.h"
 #include "third_library_const.h"
-#include "graph.h"
+#include "../graph/graph.h"
 
 #ifdef __cplusplus
 /* if C++, define the rest of this header file as extern C */
